rsmtool/rsmexplain.py

```diff
@@ -0,0 +1,525 @@
+#!/usr/bin/env python
+"""
+Explain a SKLL model using SHAP explainers.
+
+:author: Remo Nitschke (rnitschke@ets.org)
+:author: Zhaoyang Xie (zxie@etscanada.ca)
+:author: Nitin Madnani (nmadnani@ets.org)
+
+:organization: ETS
+"""
+
+import glob
+import logging
+import os
+import pickle
+import sys
+import warnings
+from os import listdir
+from os.path import abspath, basename, exists, join, normpath, splitext
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+# TODO: remove this context manager around shap import when switching to
+# new version of SHAP that is properly numba-compliant
+with warnings.catch_warnings(record=False) as w:
+    warnings.filterwarnings("ignore", message=".*The 'nopython' keyword.*")
+    import shap
+
+from skll.data import FeatureSet
+
+from .configuration_parser import configure
+from .modeler import Modeler
+from .preprocessor import FeaturePreprocessor
+from .reader import DataReader
+from .reporter import Reporter
+from .utils.commandline import ConfigurationGenerator, setup_rsmcmd_parser
+from .utils.constants import VALID_PARSER_SUBCOMMANDS
+from .utils.conversion import parse_range
+from .utils.logging import LogFormatter
+
+
+def select_features(featureset, range_size=None):
```

Comment:
The name is confusing since no feature selection is done. Rename to something like `get_subset`?